test/e2e/sampleapptestbase.go

```diff
@@ -157,7 +157,8 @@ func checkDeployment(t *testing.T, appName, expectedOutput string) {
 		t.Fatalf("Ingress not found (ingress='%s', host='%s')", ingressAddr, serviceHost)
 	}
 	t.Logf("Curling %s/%s", ingressAddr, serviceHost)
-
+	
+	serviceHost = strings.Replace(serviceHost, "http://", "", 1)
```

Comment:
Since this used to take domain name, maybe also need to strip out path from the url?

Reply:
In general: you can parse it as a `net/url.URL` and extract the `.Host` for the host header below, and replace the `Host` with the `ingressAddr` to curl to.

This is good enough for anything we'd emit today.